MatrixKit/Categories/NSBundle+MatrixKit.m

```diff
@@ -15,16 +15,21 @@
  */
 
 #import "NSBundle+MatrixKit.h"
+#import "MXKViewController.h"
 
 @implementation NSBundle (MatrixKit)
 
 static NSString *customLocalizedStringTableName = nil;
 
 + (NSBundle*)mxk_assetsBundle
 {
-    NSString *bundleResourcePath = [NSBundle mainBundle].resourcePath;
-    NSString *assetPath = [bundleResourcePath stringByAppendingPathComponent:@"MatrixKitAssets.bundle"];
-    return [NSBundle bundleWithPath:assetPath];
+    NSBundle * bundle = [NSBundle bundleForClass:[MXKViewController class]];
```

Comment:
Your last suggestion failed when MatrixKit is integrated as a Pod in a non swift project.
The right mxk_assetsBundle implementation should be:
```
+ (NSBundle*)mxk_assetsBundle
{
    NSBundle *bundle = [NSBundle bundleForClass:[MXKViewController class]];
    NSURL *assetsBundleURL = [bundle URLForResource:@"MatrixKit" withExtension:@"bundle"];
    
    // Check whether MatrixKit is not integrated as a Pod
    if (!assetsBundleURL)
    {
        assetsBundleURL = [bundle URLForResource:@"MatrixKitAssets" withExtension:@"bundle"];
    }
    
    return [NSBundle bundleWithURL:assetsBundleURL];
}
```
Please update your PR and test it on your side, this should be ok now ;)